tests/languages/clojure/comment_feature.test

```diff
@@ -0,0 +1,15 @@
+;; this is comment
+; this is another comment
+; this is another comment
```

Comment:
Those two are exactly the same, aren't they? There is no need to test for the exact same thing twice.